com.spotify.json

```diff
@@ -0,0 +1 @@
+{"name":"com.spotify","author":"hkini@thingpedia.stanford.edu","main":"index.js","thingpedia-version":38}
```

Comment:
You should add your name as author too!

Also you can remove the thingpedia-version field.